package/com.unity.formats.usd/Tests/Common/CustomAsserts/ExportAssert.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+using NUnit.Framework;
+
+namespace Unity.Formats.USD.Tests
+{
+    public static class ExportAssert
+    {
+        public static class Runtime
```

Comment:
I don't think this extra layer of breakdown is necessary, also because nothing in this function is specific to runtime?

Reply:
You have a good point, I moved this new test to the EditMode test, but expanding on that, I think we can move a lot of export and import tests to EditorMode tests as well